packages/lodestar/src/db/api/beacon/seenAttestationCache.ts

```diff
@@ -18,22 +18,22 @@ export class SeenAttestationCache {
     this.cache = new Map();
   }
 
-  public async addCommitteeAttestation(attestation: Attestation): Promise<void> {
+  public addCommitteeAttestation(attestation: Attestation): void {
```

Comment:
I guess this all was async to allow easier change in implementation

Reply:
The chain API was made sync where possible too. *When* we change the implementation this can become async.